browser-test/src/applicant/navigation/application_navigation_with_eligibility_conditions.test.ts

```diff
@@ -428,24 +428,6 @@ test.describe('Applicant navigation flow', () => {
         })
       })
 
-      test('Shows may be eligible alert on block edit page with an eligible answer', async ({
```

Comment:
Question: why are you removing this test? I'm not seeing the relation to the rest of the PR. Also I'm confused because I don't see a screenshot called "north-star-eligible-alert" 🤔

Reply:
Thought: it seems like removing this functionality and updating styles are (relatively?) independent changes, so it would be easier to review as separate PRs ideally.

Reply:
Fair point, I created a separate PR for deleting the alert: https://github.com/civiform/civiform/pull/8089